docs/platform/howto/manage-domains.rst

```diff
@@ -0,0 +1,87 @@
+Manage domains
+===============
+
+Verified domains let you manage users in your organization and set up security policies.
+
+There are two ways you can verify a domain:
+
+* by adding a DNS TXT record to the domain (recommended)
+* by uploading an HTML file to the domain's root folder
+
+.. note::
+    After adding a domain, your organization users will automatically become managed users. Managed user features will be available as part of a future release.
+
+Add a domain using a DNS TXT record
+-------------------------------------
+
+#. In the organization you want to add a domain to, click **Admin**.
+
+#. Click **Domains**.
+
+#. Click **Add domain**.
+
+#. Enter a **Domain name**.
+
+#. In the **Verification method**, select **Add a DNS TXT record to your domain host**.
+
+#. In another browser tab or window, log in to your domain hosting provider.
+
+#. In the domain hosting provider, go to the DNS settings.
+
+#. In the Aiven Console, copy the TXT record value.
+
+#. In the DNS settings, create the TXT record with the following:
+
+.. list-table::
+      :header-rows: 1
+      :align: left
+
+      * - Field name
+        - Value
+      * - Name
+        - ``_aiven-challenge.{your domain}``
+      * - Value 
+        - The TXT record value shown in Aiven Console.
+      * - Type
+        - ``TXT``
+
+#. In the Aiven Console, click **Add domain**.
+
+It can take up to 72 hours for your DNS records to update the domain to be verified.
+
+
+Add a domain using an HTML file 
+--------------------------------
+
+#. In the organization that you want to remove a domain from, click **Admin**.
+
+#. Click **Domains**.
+
+#. Click **Add domain**.
+
+#. Enter a **Domain name**.
+
+#. In the **Verification method**, select **Upload an HTML file to your website**.
+
+#. Download the HTML file.
+
+#. In another browser tab or window, upload the HTML file to your website's root directory.
```

Comment:
In this case we probably don't have to mention "In another browser or window", depending on the customer size and their technical processes this flow will be drastically different, but I think most commonly it will include some kind of version control system like git for example.
Also, it would be useful to have the actual path we want it to be, so something like "upload the HTML file to your website in the path <domain>/.well-known/aiven/<html_file>".